include/deal.II/multigrid/mg_transfer_global_coarsening_util.h

```diff
@@ -0,0 +1,75 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_mg_transfer_global_coarsening_util_h
+#define dealii_mg_transfer_global_coarsening_util_h
+
+#include <deal.II/lac/affine_constraints.h>
+#include <deal.II/lac/la_parallel_vector.h>
+
+#include <deal.II/multigrid/mg_base.h>
+#include <deal.II/multigrid/mg_transfer_global_coarsening.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace MGTransferUtilities
+{
+  /**
+   * Check if the polynomial transfer between @p fe_degree_fine and
+   * @p fe_degree_coarse is supported.
+   *
+   * @note Currently, the polynomial coarsening strategies: 1) go-to-one,
+   *   2) bisect, and 3) decrease-by-one is supported.
+   */
+  bool
+  polynomial_transfer_supported(const unsigned int fe_degree_fine,
```

Comment:
Based on the documentation, I have no idea what this function does. Would this be more appropriate as a static member of MGTransferTwoLevel? I assume that this is what the function reports on.